crates/jstz_cli/src/logs/mod.rs

```diff
@@ -1,5 +1,7 @@
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 use clap::Subcommand;
+use jstz_crypto::public_key_hash::PublicKeyHash;
+use jstz_proto::context::account::Address;
```

Comment:
Use of both `PublicKeyHash` and `Address` is a code smell (since `Address` should be an alias of `PublicKeyHash`)